articles/quickstart.md

```diff
@@ -343,24 +350,46 @@ let res = M(qubit);
 
 Now the results get more interesting:
 
-```Output
-Init:Zero 0s=484  1s=516
-Init:One  0s=522  1s=478
+```dotnetcli
+dotnet run --count 1000 --initial One
+Test results (# of 0s, # of 1s):
+(496, 504)
+
+dotnet run --count 1000 --initial Zero
+Test results (# of 0s, # of 1s):
+(506, 494)
 ```
 
-Every time we measure, we ask for a classical value, but the qubit is halfway between 0 and 1, so we get (statistically) 0 half the time and 1 half the time. This is known as __superposition__ and gives us our first real view into a quantum state.
+Every time we measure, we ask for a classical value, but the qubit is halfway
+between 0 and 1, so we get (statistically) 0 half the time and 1 half the time.
+This is known as **superposition** and gives us our first real view into a
+quantum state.
```

Comment:
```suggestion
This is known as **superposition** and gives us our first real look at the uniqueness of quantum states.
```
This specific rewording is just an idea, but I think the clarification should be there.
Similar to above regarding the risk of referring to the prior stuff as *being* classical, we're barreling toward confusion if we imply that the computational basis states are *not* quantum states.